src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -392,8 +392,18 @@ private LoadedCvrData parseCastVoteRecords(ContestConfig config, Progress progre
       progress.markFileRead();
     }
 
-    // Output the RCTab-CSV CVR
-    if (!encounteredSourceProblem) {
+    if (encounteredSourceProblem) {
```

Comment:
Looks like you've got duplicate code between here and line 420.

Reply:
Bad merge! Fixed, thanks for catching.